src/combinatorics/simple_graph/clique.lean

```diff
@@ -123,6 +137,47 @@ def clique_free (n : ℕ) : Prop := ∀ t, ¬ G.is_n_clique n t
 
 variables {G H}
 
+lemma clique_free_iff (n : ℕ) :
+  G.clique_free n ↔ is_empty ((⊤ : simple_graph (fin n)) ↪g G) :=
```

Comment:
```suggestion
lemma clique_free_iff {n : ℕ} :
  G.clique_free n ↔ is_empty ((⊤ : simple_graph (fin n)) ↪g G) :=
```
because it's an iff.

The proof is also a bit worryingly long.

Reply:
I split it up into parts that are hopefully less worrying.